internal/signerverifier/ssh/ssh.go

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: Apache-2.0
+package ssh
+
+import (
+	"bytes"
+	"context"
+	"crypto"
+	"encoding/base64"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/hiddeco/sshsig"
+	"golang.org/x/crypto/ssh"
+)
+
+// FIXME: make this the interface for Keys in TUF metadata
+type Key struct {
+	KeyType string
```

Comment:
Do we just drop this in favour of scheme if we're stripping out go-sslib after this lands?

Reply:
I think we could. I'm currently assigning the value `ssh`, but I could also add that to the scheme (if necessary).